src/main/java/com/iemr/common/controller/users/IEMRAdminController.java

```diff
@@ -944,7 +954,17 @@ public String forceLogout(@RequestBody ForceLogoutRequestModel request, HttpServ
 	    }
 	    return outputResponse.toString();
 	}
-	
+	private String getJwtTokenFromCookies(HttpServletRequest request) {
```

Comment:
Look up for current token seems to be getting repeated in so many places. Can we make this a reusable method?